src/components/SingleSourceRow.vue

```diff
@@ -21,110 +21,76 @@
 </script>
 
 <template>
-    <article class="card card-single card_single-source-row"  v-if="rowObject">
-      <header class="card-header">
-        <h2 class="card-title">Dig In</h2>
-      </header>
-      <section class="card-content">
-        <article class="card-tile" v-for="(value, key) in rowObject" :key="key">
-          <a class="article-anchor-wrapper" :href="value?.article_url">
-            <span class="article-image-wrapper" :style="{ 'background-image': 'linear-gradient(0deg, rgba(0,0,0,0) 0%, rgba(46,46,46,0.4) 35%, rgba(0,0,0,0.7) 100%), url(' + value?.article_image + ')' }">
-              <img class="article-logo" :src="value?.article_logo" alt="cows">
-              <h5 class="article-title">{{ value?.article_title }}</h5>
-            </span>
-          </a>
-        </article>
-      </section>
+
+  <div class="swipe-wrapper" v-if="rowObject">
+    <article class="card-tile" v-for="(value, key) in rowObject" :key="key">
+      <a class="article-anchor-wrapper" :href="value?.article_url">
+        <span class="article-image-wrapper" :style="{ 'background-image': 'linear-gradient(0deg, rgba(0,0,0,0) 0%, rgba(46,46,46,0.4) 35%, rgba(0,0,0,0.7) 100%), url(' + value?.article_image + ')' }">
+          <img class="article-logo" :src="value?.article_logo" alt="cows">
+          <h5 class="article-title">{{ value?.article_title }}</h5>
+        </span>
+      </a>
     </article>
+  </div>
+
 </template>
 
 <style scoped lang="scss">
 
-  .card {
+  .swipe-wrapper {
 
-    position: relative;
+    padding: $card-padding-internal;
 
-    .card-header,
-    .card-footer {
-      position: relative;
-      z-index: 20;
+    // todo: unsure if this'll stay. probably do a smarter implementation later.
+    &::-webkit-scrollbar {
+      display: none;
     }
 
-    &::before,
-    &::after {
-
-      content: '';
+    @include mq('large') {
 
-      display: block;
-      width: $card-padding-internal;
+      display: grid;
       height: 100%;
 
-      position: absolute;
-      top: 0;
-      bottom: 0;
+      gap: 1rem;
+      grid-auto-columns: minmax(22rem, auto);
```

Comment:
is "minmax" actually the right thing? it totally might be i dont know anything about grid, it just _looks_ like a typo

Reply:
It is A thing, if it's the RIGHT thing... who knows. It worked.